landing-pages/site/content/en/blog/airflow-2.10.0/index.md

```diff
@@ -0,0 +1,129 @@
+🐳 Docker Image: "docker pull apache/airflow:2.10.0" \
+🚏 Constraints: https://github.com/apache/airflow/tree/constraints-2.10.0
+
+
+## Hybrid Execution
+
+Each executor comes with its unique set of strengths and weaknesses, typically balancing latency, isolation, and compute efficiency. Traditionally, an Airflow environment is limited to a single executor, requiring users to make trade-offs, as no single executor is perfectly suited for all types of tasks.
+
+We are introducing a new feature that allows for the concurrent use of multiple executors within a single Airflow environment. This flexibility enables users to take advantage of the specific strengths of different executors for various tasks, improving overall efficiency and mitigating weaknesses. Users can set a default executor for the entire environment and, if necessary, assign particular executors to individual DAGs or tasks.
+
+To configure multiple executors we can pass comma separated list in airflow configuration. The first executor in the list will be the default executor for the environment.
+
+```
+[core]
+executor = 'LocalExecutor,CeleryExecutor'
+```
+To make it easier for dag authors, we can also specify aliases for executors that can be specified in the executor configuration
+```commandline
+[core]
+executor = 'LocalExecutor,my.custom.module.ExecutorClass:ShortName'
+```
+
+DAG authors can specify executors to use at the task
+```python
+BashOperator(
+    task_id="hello_world",
+    executor="LocalExecutor",
+    bash_command="echo 'hello world!'",
+)
+
+@task(executor="LocalExecutor")
+def hello_world():
+    print("hello world!")
+```
+
+We can also specify executors on the DAG level
+
+```python
+def hello_world():
+	print("hello world!")
+
+def hello_world_again():
+	print("hello world again!")
+
+with DAG(
+    dag_id="hello_worlds",
+    default_args={"executor": "LocalExecutor"},  # Applies to all tasks in the DAG
+) as dag:
+    # All tasks will use the executor from default args automatically
+    hw = hello_world()
+    hw_again = hello_world_again()
+```
+
+## Dynamic Dataset scheduling through DatasetAlias
+
+Airflow 2.10 comes with `DatasetAlias` class which can be passed as a value in the `outlets`, `inlets` on a task, and `schedule` on a DAG. An instance of `DatasetAlias` is resolved dynamically to a real dataset. Downstream can depend on either the resolved dataset or on an alias itself.
+
+`DatasetAlias` has one argument `name` that uniquely identifies the dataset. The task must first declare the alias as an outlet, and use `outlet_events` or `yield Metadata` to add events to it.
+
+### Emit a dataset event during task execution through outlet_events
+```python
+from airflow.datasets import DatasetAlias
+
+@task(outlets=[DatasetAlias("my-task-outputs")])
+def my_task_with_outlet_events(*, outlet_events):
+    outlet_events["my-task-outputs"].add(Dataset("s3://bucket/my-task"), extra={"k": "v"})
```

Comment:
```suggestion
    outlet_events["my-task-outputs"].add(Dataset("s3://bucket/my-task"), extra={"k": "v"})
```

I'd get rid of the extra here, it's just extra noise as it isn't related to DatasetAlias.